collection_requirements.rst

```diff
@@ -0,0 +1,139 @@
+  * If the Collection works with Ansible 2.9, then this should be set to `>=2.9.10`
+
+Modules & Plugins
+------------------
+
+Documentation
+~~~~~~~~~~~~~~
+All module and plugin ``DOCUMENTATION`` and ``RETURN`` MUST:
+
+* Use the FQCN for ``M(...)`` See `Linking within module documentation<https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_documenting.html#linking-within-module-documentation>`_
+* Use the FQCN for ``seealso`` FIXME Docs link
+
+All module and plugin ``EXAMPLES`` MUST:
+
+* Use FQCN for module (or plugin) name.
+
+Other items:
+* You MUST Use the FQCN for ``extends_documentation_fragment:``, unles you are referring to doc_fragments from ansible-base
+
+
+Contributor Workflow
+====================
+
+Changelogs
+~~~~~~~~~~
+
+To give a consistent feel for changelogs across collections, and ensure for collections included in the ``ansible`` package we suggest you use `antsibull-changelog <https://github.com/ansible-community/antsibull-changelog>`_
+
+
+Preferred (in descending order):
+
+1. Use antsibull-changelog (preffered)
+2. Provide ``changelogs/changelog.yaml`` in the `correct format <https://github.com/ansible-community/antsibull-changelog/blob/main/docs/changelog.yaml-format.md>`_
+3. Provide a link to the changelog file (self-hosted) (not recommended)
+
+Versioning and deprecation
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+FIXME Details here
+* Deprecation vs Collection version
+* How many Collection major versions to give
+
+Repo management
+===============
+
+Repo name
+~~~~~~~~~
+
+For collections under ansible-collections the repo SHOULD be named ``NAMESPACE.COLLECTION``
+
+Branch name and configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+All new repos under `ansible-collections<https://github.com/ansible-collections>`_ MUST have `main`` as the default branch.
+
+Existing repos SHOULD be converted to use ``main``
+
+Repo Protections:
+* Allow merge commits: disallowed
+
+Branch protections MUST be enforced
+* Require linear history
+* Include administrators
+
+CI Testing
+===========
+
+At a minimim ``ansible-test sanity`` MUST be run from the `latest stable ansible-base branch <https://github.com/ansible/ansible/branches/all?query=stable->`_
+
+For most repos GitHub actions are sufficient, see `example<https://github.com/ansible-collections/collection_template/tree/main/.github/workflows>`_
+
+FIXME to write a guide "How to write CI tests" (from scratch / add to existing) and put the reference here
+
+Unit Testing
+============
+
+
+Collections and Working Groups
+==============================
+* Working group page(s) on a corresponding wiki (if needed. Makes sense if there is a group of modules for working with one common entity, e.g. postgresql, zabbix, grafana, etc.)
+* Issue for agenda (or pinboard if there aren't regular meetings) as pinned issue in the Repo
+
+When moving modules between collections
+=======================================
```

Comment:
```suggestion
When moving modules between collections
=======================================
```
After @samccann 's pr https://github.com/ansible/ansible/pull/70243 is merged, would be good to have
the reference to the fixed related part of the doc (and maybe the reference to
https://github.com/ansible-collections/overview/blob/master/README.rst#q-what-should-i-do-to-move-plugins-across-collections-during-migration)

Reply:
Good call, added a note on https://github.com/ansible/ansible/pull/70243#issuecomment-666480846